tests/deep-copy/pass_lee.js

```diff
@@ -1,5 +1,6 @@
+const isObj = (x) => x && (x.constructor === Object || !x.constructor)
 const deepCopy = (target) => {
-  var result = Array.isArray(target) ? [] : {}
+  var result = Array.isArray(target) ? [] : isObj(target) ? {} : target
```

Comment:
why `var` ?
Also this code will crash with `undefined`

We should add an `undefined` value in our tests haha

Reply:
i tried to run with an `undefined` value, like: `deepCopy(Object.freeze({ undef: undefined })).undef` and the output is correct `undefined`